scripts/json2db.pl

```diff
@@ -141,17 +141,23 @@ sub fetch_policies {
     # so can be just be simply stored for reference in policy page.
     my $out_agreements = {};
 
-    foreach my $policy (@{$policies->{policies}}) {
+    foreach my $policy (@$policies) {
         my $policy_id = $policy->{"id"};
         foreach my $agreement (@{$policy->{"agreement_links"}}) {
             my $decision = $agreement->{"decision"};
-            my $chamber = $decision->{"chamber"};
+            my $decision_date = $decision->{"date"};
+            # adjust decision_ref - split by . and remove the last part and put it back together
+            # this is because the decision ref is to a specific line, rather than a linkable section.
+            my @parts = split(/\./, $decision->{"decision_ref"});
+            pop @parts if @parts > 1;
+            my $decision_ref = join(".", @parts);
```

Comment:
I feel like maybe calling this `$decision_url_ref` would be a bit clearer or `$division_section_ref` maybe?